core/src/main/scala/kafka/server/ClientMetricsManager.scala

```diff
@@ -0,0 +1,38 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package kafka.server
+
+import kafka.metrics.clientmetrics.ClientMetricsConfig
+
+import java.util.Properties
+
+object ClientMetricsManager {
+  private val Instance = new ClientMetricsManager
+
+  def getInstance: ClientMetricsManager = Instance
```

Comment:
@hachikuji @mjsax @AndrewJSchofield Inherited some classes from earlier implementation of KIP and not sure if I should properly define trait for the client telemetry classes in broker.

Reply:
Moved to java